org.virt_manger.virt-viewer.appdata.xml

```diff
@@ -0,0 +1,60 @@
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.virt_manager.virt-viewer</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Remote Viewer</name>
+  <!-- From the GNOME Software utility -->
+  <summary>A remote desktop client built with GTK-VNC, SPICE-GTK and libvirt</summary>
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apa.html -->
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apas02.html -->
+  <categories>
+    <category>Utility</category>
+    <categories>Network</categories>
+  </categories>
+  <url type="homepage">https://virt-manager.org/download/</url>
+  <url type="bugtracker">https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&bug_status=ASSIGNED&classification=Community&component=virt-viewer&product=Virtualization%20Tools</url>
+  <url type="help">https://support.mozilla.org/questions/thunderbird</url>
```

Comment:
Wrong help URL.

Reply:
Thanks for catching this, I wasn't really able to find a good "help" URL, so I'll just remove that line. Is that okay?

Reply:
Sure.